slackbridge/messages.py

```diff
@@ -55,7 +57,52 @@ def resolve(self):
         if not channel_id or not isinstance(channel_id, str):
             return
 
-        if channel_id in self.bridge_bot.channels:
+        if channel_id[0] == 'D':  # DM channels start with a D
+            if not user_bot.im_id:
+                user_bot.im_id = channel_id
+
+            if 'text' in self.raw_message:
+                match = re.search('(.*\w):', self.raw_message['text'])
+                if match:
+                    rcpt = match.group(1)
+                    if rcpt in self.bridge_bot.irc_users and self.deferred:
+                        irc_user = self.bridge_bot.irc_users[rcpt]
+                        if irc_user.authenticated:
+                            self.raw_message['text'] = \
```

Comment:
For clarity I'd do
```python
msg = self.raw_message['text']
msg.replace(match.group(0), '').strip()
self.raw_message['text'] = msg
```
rather than using the `\`